src/screens/DomainScreen/elements/Header/elements/ActionButtonGroup.js

```diff
@@ -105,7 +105,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     borderRadius: 8,
     backgroundColor: COLORS.signed_primary,
-    color: COLORS.white
+    color: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the button color might affect readability if the background color is dark. Please ensure that this color change maintains sufficient contrast for accessibility.

```diff
-    color: COLORS.white
+    color: COLORS.almostBlack
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->